### ISSUE TITLE
Give `external` higher billing in the docs?

### ISSUE THREAD

**Issue by ivirshup** [scverse/scanpy#588]
At the moment external modules are kind of hidden in the docs. I think it'd be worth making them more visible (at least on the same page as everything else). I've been giving this a shot, but have hit the limit of my sphinx/ rst abilities.

Two ideas for how they could be more discoverable:

* They get their own heading under `api`
* They're mixed in with everything else (so everything stays organized by topic), but their names are prepended with `sce` while scanpy functions are prepended with `sc`.

**Comment by falexwolf** [comment]
Hm, they're mentioned in the note at the very top, and there are cross-references within the page. They have a separate page just as plotting.

I'm happy to give them a separate heading, just like plotting has. I wouldn't want them to be mixed in. I expect that list of external tools to potentially become quite long at some point. I'd also prefer them to stay on a separate page.

**Comment by ivirshup** [comment]
I've got two main reasons for thinking they should be more visible:

1. If I'm trying to find what tools are available through scanpy for a certain task, it should be very obvious where that might be available.  For example, if I want to know what's available for batch correction, I (the user) am probably not too fussed about whether it's in the scanpy codebase or not.
2. As a method developer, it'd encourage me to integrate my method if I saw it'd be highly visible and that other people were doing it. Right now there are links, but users still have to go to see the notes with those links, go to a separate page, and scroll for a bit to see any particular method. 

Another strategy could be a top level `External API` heading underneath the `API` heading? Then there could be an expandable table of contents (how I typically navigate the site) to get an idea of what's there.
